modules/oracle-xe/src/main/java/org/rnorth/testcontainers/containers/OracleContainer.java

```diff
@@ -0,0 +1,90 @@
+package org.rnorth.testcontainers.containers;
+
+import com.spotify.docker.client.messages.ContainerConfig;
+import com.spotify.docker.client.messages.ContainerInfo;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import com.zaxxer.hikari.pool.PoolInitializationException;
+
+/**
+ * Created by gusohal on 26/04/15.
+ */
+public class OracleContainer extends AbstractContainer implements DatabaseContainer  {
+
+    private static final String IMAGE = "alexeiled/docker-oracle-xe-11g";
+    private String oraclePort;
+
+    @Override
+    protected void containerIsStarting(ContainerInfo containerInfo) {
+        oraclePort = containerInfo.networkSettings().ports().get("1521/tcp").get(0).hostPort();
+    }
+
+    @Override
+    protected String getLivenessCheckPort() {
+        return oraclePort;
+    }
+
+    @Override
+    protected ContainerConfig getContainerConfig() {
+        return ContainerConfig.builder()
+                .image(getDockerImageName())
+                .exposedPorts("1521", "22", "8080")
```

Comment:
Just curious, is it necessary to expose port 22? I imagine port 8080 is exposed for the db Home Page for administration, but I'm wondering if ssh access would be as useful.

No big deal, just curious!